README.rst

```diff
@@ -419,6 +419,10 @@ The following flags are accepted:
 |                                                                                                                                                         |
 | The ``repository_macro`` directive should be added to the WORKSPACE in order for future Gazelle calls to recognize the repos defined in the macro file. |
 +----------------------------------------------------------------------------------------------------------+----------------------------------------------+
+| :flag:`-prune true|false`                                                                                | :value:`false`                               |
++----------------------------------------------------------------------------------------------------------+----------------------------------------------+
+| When true Gazelle will remove `go_repository`_ rules that no longer have equivalent repos in the ``Gopkg.lock``/``go.mod`` file.                        |
```

Comment:
Add comma: "When true, Gazelle..."

Mention that this only works with `-from_file`.